### PR TITLE
Upgrade mafia

### PR DESCRIPTION
**Pull request by jystic**

### CODE REVIEW

**Comment by nhibberd** [comment]
Please use the latest mafia only

File: mafia [hunk 1]
```diff
@@ -52,12 +87,12 @@ exec_mafia () {
...
-      mkdir -p $(dirname $MAFIA_PATH)
-      cp "$MAFIA_TEMP/mafia/.cabal-sandbox/bin/mafia" "${MAFIA_PATH}.tmp"
-      mv "${MAFIA_PATH}.tmp" "$MAFIA_PATH"
+      cp "$MAFIA_TEMP/mafia/.cabal-sandbox/bin/mafia" "$MAFIA_PATH_TEMP"
+      chmod +x "$MAFIA_PATH_TEMP"
+      mv "$MAFIA_PATH_TEMP" "$MAFIA_PATH"
     }
 
-    exec $MAFIA_PATH $@
+    exec $MAFIA_PATH "$@"
```
Comment (amosr) [inline mafia]: eyy! does this fix the arguments not being passed to mafia bug I seem to run into sometimes?

**Comment by amosr** [comment]
:+1: nobody ever got fired for using the latest mafia

File: mafia (hunk 1 above)
Comment (jystic) [inline mafia]: maybe so! give it a try, it could be a problem with how i'm slurping them in via optsparse though

**Pull request merged**